resources/config_custom.toml

```diff
@@ -1,8 +1,5 @@
 [aion]
-chain = "core/res/aion/custom.json"
-#keys_path = None
-#base_path = None
-#db_path = None
```

Comment:
The comments means the option's value is 'None' (because rust cannot parse the 'None' value in the toml file). But it has these options to customize. Recommended to keep comments.